src/index.ts

```diff
@@ -151,15 +160,19 @@ export default class FileOrganizer extends Plugin {
     content: string,
     name: string
   ): Promise<{ type: string; formattingInstruction: string } | null> {
+    if (!this.settings.aiTemplateFormatting) {
```

Comment:
this can't work as expected since it will also prevent people to use the customclassifier from the sidebar the toggle is only meant for the inbox functionality, since it is more destructive

Reply:
addressed